src/ucp/wireup/wireup.c

```diff
@@ -301,7 +483,9 @@ static ucs_status_t ucp_wireup_send_am(ucp_ep_h ep, uct_ep_h uct_ep, uint16_t fl
      * Allocate buffer for active message.
      * TODO use custom pack callback to avoid this allocation and memcpy
      */
-    total_len = sizeof(*msg) + UCP_PEER_NAME_MAX + addr_len + aux_addr_len;
+    total_len = sizeof(*msg) +
```

Comment:
total_len here will be 183 and then on line 549 it will fail because the max bcopy size for UGNI UDT is 128.